### ISSUE TITLE
Error with pip dependency `tenacity`

### ISSUE THREAD

**Issue by Mutantpenguin** [alyf-de/erpnext_germany#28]
After installing it into a frappe 14 environment, I got the following error:
```bash
$ /home/frappe/frappe-bench/env/bin/python -m pip install --quiet --upgrade -e /home/frappe/frappe-bench/apps/erpnext_germany
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
frappe 14.76.1 requires tenacity~=8.0.1, but you have tenacity 8.2.3 which is incompatible.
```

If version 8.2.3 really is necessary, you should try to get frappe to update it's version and remove your own dependeny on tenacity. If frappes version 8.0.1 is enough, please remove your own.

### Versions
```
erpnext 14.70.4
erpnext_germany 14.6.1
frappe 14.76.1
```

**Comment by barredterra** [comment]
Should be resolved via 51c1f6a0a2b19e99e0f4c96aa9d080d13a5e67de

**Comment by Mutantpenguin** [comment]
great, thank you very much
